setup.py

```diff
@@ -19,13 +19,17 @@
     linetrace = True
     extra_args.append(('CYTHON_TRACE_NOGIL', '1'))
 
+extra_compile_args = []
+if platform.system() == 'Windows':
```

Comment:
Shouldn’t this be `!=` to actually match the old condition?

Reply:
Oops, done